build/nuget-org.packages.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should probably give the file a different extension to differentiate from the file with the same name ends up in under the release nugets folder in our drops (that file also additionally contains version numbers for the packages that are built).

May I suggest `nugget-org.pkglist`?